lib/registrant.rb

```diff
@@ -0,0 +1,57 @@
+require 'pry'
+
+class Registrant
+  attr_reader :name, 
+              :age, 
+              :permit, 
```

Comment:
Careful here, we do not want a method called `.permit` exposed. We have a method called `.permit?` so no need for this in the attr_reader.